src/Employer.Web/wwwroot/css/application.css

```diff
@@ -496,4 +496,9 @@ li.menu-link.sub-menu-open ul li a {
 
 .actions .action {
   margin-bottom: 20px;
+}
+
+/***  Inline form ***/
+.form-inline{
```

Comment:
🎨 - Is this because of that inline style CSP error?

Reply:
Nope this is because the user-agent by default renders <form> as a block element.  I want to hide the whole form if a vacancy is already submitted (so the form and submit button are never rendered) but by doing this the styling was screwing up.